pkg/commands/test/test.go

```diff
@@ -33,7 +33,7 @@ var (
 )
 
 // NewTestCommand creates a new test command
-func NewTestCommand() *cobra.Command {
+func NewTestCommand(osExit func(int)) *cobra.Command {
```

Comment:
I think it would be a little cleaner if we returned a `*cobra.Command` like this instead,
```go
	cmd := &cobra.Command{
		Use:     "test <file> [file...]",
		Short:   "Test your configuration files using Open Policy Agent",
		Version: fmt.Sprintf("Version: %s\nCommit: %s\nDate: %s\n", constants.Version, constants.Commit, constants.Date),
		RunE: func(cmd *cobra.Command, args []string) error {
			// ...
		},
	}
```